common/repositories/buyer_repository.h

```diff
@@ -106,13 +106,8 @@ class BuyerRepository: public BaseBuyerRepository {
 				return false;
 			}
 
-			auto buy_lines = BaseBuyerBuyLinesRepository::GetWhere(
-				db,
-				fmt::format("`buyer_id` = '{}'", buyer.front().id)
-			);
-			if (buy_lines.empty()) {
-				return false;
-			}
+			auto buy_lines =
```

Comment:
```cpp
auto buy_lines = BaseBuyerBuyLinesRepository::GetWhere(
	db, 
	fmt::format("`buyer_id` = '{}'", buyer.front().id)
);
```

Reply:
Is the id an integer? We shouldn't quote integers because it can throw off the query planner and miss indexes

Reply:
It is, I update all the references.  Will watch for others as well.